src/lib.rs

```diff
@@ -117,7 +117,7 @@ fn process_word(word: &str) -> Cow<'_, str> {
     if SMALL_RE.is_match(&lower_word) {
         Cow::from(lower_word)
     } else if starts_with_bracket(word) {
-        let rest = titlecase(&word[1..]);
+        let rest = titlecase(&word.chars().skip(1).collect::<String>());
```

Comment:
The slice in this case is guarded by `starts_with_bracket` so it can't cause a panic because we only get in here if the first character is `(`.